src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -63,11 +63,14 @@ public function testLoadFile()
 
     public function testLoadFileWithInternalErrorsEnabled()
     {
-        libxml_use_internal_errors(true);
+        $internalErrors = libxml_use_internal_errors(true);
 
         $this->assertSame(array(), libxml_get_errors());
         $this->assertInstanceOf('DOMDocument', XmlUtils::loadFile(__DIR__.'/../Fixtures/Util/invalid_schema.xml'));
         $this->assertSame(array(), libxml_get_errors());
+
+        libxml_clear_errors();
```

Comment:
You are right, I'll make a note to send a new PR :)

However I don't think it is a very big problem, given that if an assert fails or exception is thrown the changes are pretty big someone will try again after fixing the code anyway (,but again you are absolutely right).